controllers/main_controller.rb

```diff
@@ -9,3 +9,19 @@
   # combined document back to the client.
   erb :"main/welcome"
 end
+
+MyApp.get "/main/about" do
```

Comment:
This controller seems to duplicate the work being done by other controllers. You only need to define a particular controller action once.